zahra-portfolio.html

```diff
@@ -0,0 +1,241 @@
+                        <li><button class="social-button" type="button"><img src="images/twitter.svg" alt="twitter"></button></li>
+                        <li><button class="social-button" type="button"><img src="images/Linkedin.svg" alt="linkedin"></button></li>
+                        <li><button class="social-button" type="button"><img src="images/m.svg" alt="m"></button></li>
+                        <li><button class="social-button" type="button"><img src="images/git.svg" alt="git"></button></li>
+                        <li><button class="social-button" type="button"><img src="images/Vector.svg" alt="vector"></button></li>
+                    </ul>
+                  </div>
+                  <div class="action2">
+                <button type="button" class="button">Get My Resume</button>
+              </div>
+          </div>
+        
+          <div class="skill">
+            <div class="listfram">
+                <div class="listtag ">
+                    <p class="listname">Langguages</p>
+                    <img src="images/Disabled-right.svg" class="disabled" alt="button">
+                    <img src="images/Disabled-down.svg" class="enabled" alt="button">
+                </div>
+
+                <ul class="dropdown-content">
+                    <li>
+                        <div class="listimg"><img src="images/js.svg" alt="javascript"></div>
+                        <p class="listtxt">JavaSript</p>
+                    </li>
+                    <li>
+                        <div class="listimg"><img src="images/html.svg" alt="html"></div>
+                        <p class="listtxt">HTML</p>
+                    </li>
+                    <li>
+                        <div class="listimg"><img src="images/css.svg" alt="CSS"></div>
+                        <p class="listtxt">CSS</p>
+                    </li>
+                </ul>
+              </div>
+                <div class="listfram ">
+                    <div class="listtag ">
+                        <p class="listname">Frameworks</p>
+                        <img src="images/Disabled-right.svg" class="disabled" alt="button">
+                    <img src="images/Disabled-down.svg" class="enabled" alt="button">
+                    </div>
+                    <ul class="skillul">
+                    </ul>
+                </div>
+                <div class="listfram ">
+                    <div class="listtag ">
+                        <p class="listname">Skills</p>
+                        <img src="images/Disabled-right.svg" class="disabled" alt="button">
+                    <img src="images/Disabled-down.svg" class="enabled" alt="button">
+                    </div>
+                    <ul class="skillul">
+                    </ul>
+                </div>
+
+            
+        </div>
+       
+        </section>
+        <footer>
+          <section class="wrapper">
+            <h2 class="contact-title">Contact me</h2>
```

Comment:
- [ ] Contact me section has a  white line background visible on both sides and footer part:
![image](https://user-images.githubusercontent.com/77038610/150353940-272d6770-fdba-4cdd-8575-c1438fad595b.png)

- [ ]  please kindly make the `purple` color contact me section spread all along the contact section like it was given in figma 😉